### DISCUSSION TITLE
付费后还是无法升级专业版，邮箱也无效。

### DISCUSSION THREAD

**Discussion by xilijuli** [getlantern/lantern-forum-cn#795] [General]
![QQ图片20240818100029](https://github.com/user-attachments/assets/32d27ed0-1980-460f-ab25-04529ceeab9c)
账号是345408353@qq.com。

**Comment by MaxMKMax** [comment]
@xilijuli 您好. 好的, 我已经帮您提交问题上去了. 过 1-2 天请您查看这个留言区或邮箱.

**Reply by jwj0329** [reply]
我也是付费后无法升级专业版，邮箱zjxy0329@126.com
![微信图片_20240819212016](https://github.com/user-attachments/assets/36d9551c-8e29-4814-b8f0-ce6323c05544)

**Reply by xilijuli** [reply]
账号是[345408353@qq.com](mailto:345408353@qq.com)。还是没解决呢。27个月的。

**Reply by MaxMKMax** [reply]
@xilijuli 您好. 好的. 我有帮您提交问题了. 不好意思让您久等了. 谢谢您的耐心等!
